inc/class-comment-hacks.php

```diff
@@ -150,9 +153,12 @@ public static function get_defaults() {
 	 *
 	 * @since 1.0
 	 */
-	public function set_defaults() {
-		$this->options = wp_parse_args( $this->options, self::get_defaults() );
-
-		update_option( YoastCommentHacks::$option_name, $this->options );
+	private function set_defaults() {
+		$defaults = self::get_defaults();
+		if ( ! isset( $this->options['version'] ) || YOAST_COMMENT_HACKS_VERSION > $this->options['version'] ) {
+			$this->options = wp_parse_args( $this->options, $defaults );
+			update_option( self::$option_name, $this->options );
```

Comment:
From what I can see `$this->upgrade()` will always call `update_option` at the end so I don't think this call is required.